pkg/master/master.go

```diff
@@ -586,12 +593,12 @@ func (m *Master) initV1ResourcesStorage(c *Config) {
 }
 
 // NewBootstrapController returns a controller for watching the core capabilities of the master.
```

Comment:
Updated so it's no longer a pointer. Updated the comment to describe the defaulting logic.